examples/scale/scenarios/scenarios.go

```diff
@@ -14,10 +14,50 @@
 
 package scenarios
 
-import "open-match.dev/open-match/pkg/pb"
+import (
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"open-match.dev/open-match/examples/scale/scenarios/battleroyal"
+	"open-match.dev/open-match/examples/scale/scenarios/firstmatch"
+	"open-match.dev/open-match/internal/util/testing"
+	"open-match.dev/open-match/pkg/matchfunction"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+var _ = firstmatch.Scenario()
```

Comment:
What does this achieve? A comment might help.

Reply:
Forgot to remove.